source/guides/1.1/getting-started.md

```diff
@@ -790,15 +787,17 @@ This minimal implementation should suffice to make our tests pass.
 
 ```shell
 % bundle exec rake
-Run options: --seed 63592
+Run options: --seed 27646
 
 # Running:
 
-...............
+...S.S.........
```

Comment:
Again, some auto generated tests are skipped.

Reply:
But the seed? Changing it doesn't make much sense to me.